docs/ai.md

```diff
@@ -1,33 +1,36 @@
 # AI Tools
 
-Remix has its own AI tool named **RemixAI** and a sub-project called **RemixAI Copilot** for code completion. 
+Remix has its own AI tool named **RemixAI** and a sub-project called **RemixAI Copilot** for code completion.
 
 When you load Remix, the **RemixAI Assistant** appears in the Right Side Panel.
 
 ![RemixAI Assistant Right Side Panel.](images/ai/remixai-right-panel.png)
 
-<!-- // David put the info about the minimize button here from Release 0.71
+```{tip}
+You can minimize the RemixAI Right Side Panel by clicking the minimize icon at the top left of the Right Side Panel.
```

Comment:
You can minimize the RemixAI Assistant by clicking the minimize icon at the top left of the Right Side Panel.